runtime/lang/menu_ja_jp.utf-8.vim

```diff
@@ -1,315 +1,1266 @@
-" vi:set ts=8 sts=8 sw=8 tw=0:
-"
-" Menu Translations:	Japanese (UTF-8)
-" Last Translator:	MURAOKA Taro  <koron.kaoriya@gmail.com>
-" Last Change:		20-Feb-2022.
-"
-" Copyright (C) 2001-2022 MURAOKA Taro <koron.kaoriya@gmail.com>,
-"			  vim-jp <http://vim-jp.org/>
+" Menu Translations:	  Japanese
+" Maintainer:           Ada (Haowen) Yu <me@yuhaowen.com>
+" Previous Maintainer:  MURAOKA Taro  <koron.kaoriya@gmail.com>
```

Comment:
I think that the maintainer and copyright should not be changed in this PR.

Reply:
> I think that the maintainer and copyright should not be changed in this PR.

Yes, you have a point there. How do you think copyrighted paragraph should be rewritten here?

Reply:
> I think that the maintainer and copyright should not be changed in this PR.

After applying this temple, the previous maintainer may not recognize this file. It's hard for them to continue to maintain.